### ISSUE TITLE
[BUG] New TV Shows (IMDb)

### ISSUE THREAD

**Issue by saxoloman** [umbrellaplug/umbrellaplug.github.io#361]
### Bug

I didnt know how to classify this.
its not a bug but rather a change if possible ?
In umbrella under discover TV shows ---> New TV Shows (IMDb)
a list is loaded with like 100 tv shows.
is there a way to make this list follow paginate rule ? or make it like 50 items instead of 100 ?

### Expected Result

to paginate list or reduce number of items displayed

### Steps To Reproduce

1 go to umbrella
2 open discover TV shows
3 open New TV Shows (IMDb)
4 wait for a very long list to load
5 ask umbrelladev if it can be reduced.

### Kodi Version, Type of Device, Logs, Screenshots, and Additional Info

kodi 21
coreelec

### Checklist

- [X] I have included logs and the version of Kodi I am using..
- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have verified this is not a setting.

**Comment by saxoloman** [comment]
Thats the only IMDB list that i use. But now i tried other IMDb lists and they all load ~100 results .
I guess this is normal then for IMDb lists ?
Thanks

**Comment by umbrellaplug** [comment]
All imdb lists are set this way now. Unfortunately, imdb changed their website and when you pull a list from their website now they are using a javascript button to "load more". That's stopping any type of pagination on imdb results. Most addons stopped supporting imdb and imdb lists because of this. I rewrote the scraping code to keep imdb lists alive, but it required me to set a hard limit. You can change this in the Umbrella > Tools > Settings: General

TLDR: imdb changed their website, I've done what I can to keep imdb but it required a hard limit and I cannot change that.

**Comment by saxoloman** [comment]
@umbrellaplug 
oh i see now option to reduce from 100...wish i have seen this earlier.
and thanks for explaining everything in details .

one other thing i noticed in last few versions when i open umbrella addon settings i get this in kodi log,both in windows and linux on clean install.
warning <CSettingDependencyCondition>: unable to check condition on unknown setting "!trakt.isauthed"

everything works fine,there are no problems when using addon just tough i mention it to you.

thanks for everything
take care.
